backends-velox/src/main/scala/io/glutenproject/backendsapi/velox/VeloxBackend.scala

```diff
@@ -36,17 +41,55 @@ class VeloxBackend extends Backend {
 
 object VeloxBackendSettings extends BackendSettings {
   override def supportFileFormatRead(format: ReadFileFormat,
-                                     fields: Array[StructField]): Boolean = {
+                                     fields: Array[StructField],
+                                     paths: Seq[String],
+                                     partNonEmpty: Boolean): Boolean = {
+    // Validate partition info.
+    def validatePartInfo: Boolean = {
+      // Fallback to vanilla spark when the input path does not contain the partition info.
```

Comment:
This maybe duplicated https://github.com/oap-project/gluten/pull/961/files#diff-74bb54358117b0553772ea92af4b099ccc113ed949aef65729650653f2642847R81

Reply:
Thank you. Fixed.